src/UraniumUI.Material/Controls/TextField.cs

```diff
@@ -54,6 +54,8 @@ public TextField()
         EntryView.SetBinding(Entry.IsEnabledProperty, new Binding(nameof(IsEnabled), source: this));
         EntryView.SetBinding(Entry.IsReadOnlyProperty, new Binding(nameof(IsReadOnly), source: this));
 
+        iconClear.Focused += (s, e) => { ValidateClearButtonFocus(); };
```

Comment:
Event registrations and releases should be in `OnHandlerChanged` method. Otherwise event will be stay registered and it may prevent disposing the object at the end

Reply:
Didn't know that was a thing, makes sense why your events are in there

Reply:
Good catch on that one, copy & paste magic :)